metroae

```diff
@@ -155,6 +167,130 @@ function check_run_mode {
     debug "${FUNCNAME[0]}: RUN_MODE is $RUN_MODE"
 }
 
+function phone_home {
+    if [[ $UUID == "" ]]; then
+        debug "${FUNCNAME[0]}: Skipping Phone home"
+    else
+        # append and create json data
+        json_data=""
+        while [[ $# -gt 0 ]]; do
+          json_data+="$1,"
+          shift
+        done
+        json_data=`echo $json_data | sed 's/.$//'`
+        debug "${FUNCNAME[0]}: curl -vvvv -k -H \"Content-Type: application/json\"\
+          -H \"x-apikey: $PHONE_HOME_API_KEY\"\
+          -X POST -d '{$json_data}'\
+          $PHONE_HOME_URL"
+
+        curl -vvvv -k -H "Content-Type: application/json"\
+          -H "x-apikey: $PHONE_HOME_API_KEY"\
+          -X POST -d "{$json_data}"\
+          $PHONE_HOME_URL >> /dev/null 2>&1 &
+    fi
+}
+
+function  check_if_internal_user {
+    debug ${FUNCNAME[0]}
+
+    if [[ $SKIP_UUID -eq $TRUE ]]; then
+        debug "${FUNCNAME[0]}: Using existing UUID $UUID"
+    else
+        int_user_check=`ping -c 5 github.mv.usa.alcatel.com 2>&1`
+        debug "${FUNCNAME[0]} $int_user_check"
+        SKIP_UUID=$?
+
+        if [[ $SKIP_UUID -eq $FALSE ]]; then
+            echo ""
+            write_to_screen_and_script_log "We have detected that this setup of MetroAE is internal to Nokia."
+            echo ""
+
+            declare -l continue_confirm
+            continue_confirm="init"
+            while [[ $continue_confirm != "y" ]] && [[ $continue_confirm != "n" ]] && [[ $continue_confirm != "" ]]
+            do
+                read -p " Would like to share usage data with MetroAE team? (y/N): " continue_confirm
```

Comment:
Please mention that no personally identifiable information will be collected and that we will use the information to improve the tool and instructions on how to opt out later.

Reply:
Done